### PR TITLE
[23e] Thematic indicators

### PR DESCRIPTION
**Pull request by jaklinger**
Refers to #23

NB: before running install `pip install scikit-bio` (or use the updated `requirements.txt`)

Generation of thematic indicators, via `python thematic_indicators.py` (having followed the setup and run instructions in `Pivot/README.md` followed by `indicators/two/README.md`).

Check it runs, inspect the output, review the code, check the tests run :)

Note that the indicator generation can take a couple of hours.

### CODE REVIEW

File: indicators/core/tests/topic-model-dummy-150-25/labels.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,1,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```
Comment (jaklinger) [inline indicators/core/tests/topic-model-dummy-150-25/labels.txt:1]: Dummy data for unit tests (first 10 rows of real arxiv topic tags)

File: indicators/core/tests/topic-model-dummy-150-25/most_deterministic_groups.txt [hunk 2]
```diff
@@ -0,0 +1,151 @@
+Group num., NTC
```
Comment (jaklinger) [inline indicators/core/tests/topic-model-dummy-150-25/most_deterministic_groups.txt:1]: real topic model data (tiny file) for unit testing

File: indicators/core/tests/topic-model-dummy-150-25/topics.txt [hunk 3]
```diff
@@ -0,0 +1,150 @@
+0:covid_19,sars_cov_2,coronavirus,disease,patients,infection,clinical,pandemic,covid,cancer
```
Comment (jaklinger) [inline indicators/core/tests/topic-model-dummy-150-25/topics.txt:1]: real topics (arxiv) for unit tests

**Review by Juan-Mateos — APPROVED** [review]
Need to add `scikit-bio` to the requirements. Otherwise ready to merge.

**Pull request merged**